packages/legacy/core/App/components/misc/CredentialCard.tsx

```diff
@@ -82,7 +84,14 @@ const CredentialCard: React.FC<CredentialCardProps> = ({
     }
   }
 
-  return getCredOverlayType(bundleResolver.getBrandingOverlayType())
+  const renderOpenIDCredential = () => {
+    return <OpenIDCredentialCard credentialRecord={credential as W3cCredentialRecord} onPress={onPress} />
+  }
+  if (credential instanceof W3cCredentialRecord) {
+    return renderOpenIDCredential()
```

Comment:
Do we need this `renderOpenIDCredential` function? We can just return the `OpenIDCredentialCard` component directly in the if statement, no?

Reply:
done